modules/@apostrophecms/image-widget/views/widget.html

```diff
@@ -11,8 +11,12 @@
     srcset="{{ apos.image.srcset(attachment) }}"
     src="{{ apos.attachment.url(attachment, { size: data.options.size or 'full' }) }}"
     alt="{{ attachment._alt or '' }}"
+    {% if data.options.dimensionAttrs %}
```

Comment:
This should also support adding it on the module level, so developers can only add it once if they want. That would be  `data.manager.options.dimensionAttrs`. The area configuration (what you have already) should come first to take precedence.

Reply:
The manager one is what you configured in the image widgets' `index.js` file.

Reply:
ah okay, i'll check for both like className